pyproject.toml

```diff
@@ -84,9 +82,9 @@ xml-validation = ["lxml"]
 [tool.poetry.group.dev.dependencies]
 ddt = "1.7.2"
 coverage = "7.6.1"
-flake8 = { version="7.1.2", python=">=3.8.1" }
-flake8-annotations = { version="3.1.1", python=">=3.8.1" }
-flake8-bugbear = { version="24.12.12", python=">=3.8.1" }
+flake8 = { version="7.1.2", python=">=3.10.1" }
+flake8-annotations = { version="3.1.1", python=">=3.10.1" }
```

Comment:
the bump of the py-version constraints seams a bit odd.
see https://python-poetry.org/docs/dependency-specification/#python-restricted-dependencies